tests/losses/test_giou.py

```diff
@@ -0,0 +1,23 @@
+"""
+Test Generalized Intersection over Union
+"""
+
+from unittest import TestCase
+import torch
+
+from pl_bolts.losses.giou import giou_loss
+
+
+class TestGIoULoss(TestCase):
+    def test_complete_overlap(self):
+        pred = torch.tensor([[100, 100, 200, 200]])
```

Comment:
can we test with 10, 20 instead of 100, 200?

Reply:
So I actually separated the test cases into three separate categories:
- cases where the prediction bbox and target bbox are completely overlapping (i.e. perfect prediction)
- cases where the prediction bbox and target bbox are completely non-overlapping (i.e. completely missed prediction)
- cases where the prediction bbox and target bbox are slightly overlapping

The test case you're suggesting would be slightly overlapping as opposed to completely overlapping. Do you think I should add the new test case you're suggesting or what do you think?

Reply:
I see! ok current tests are good